File: problem-set/Problem-A-Bit-Plus-Plus/BitPlusPlus.js
const bitPlusPlusString = (string) => string;
function bitPlusPlus(number, string) {
  let result = 0;

  for (let i = 0; i < number; i++) {
    let stringValue = bitPlusPlusString(string);
    if (stringValue === "X++" || stringValue === "++X") {
      result++;
    } else {
      result--;
    }
  }

  return result;
}

console.log(bitPlusPlus(2, "--X"));

File: problem-set/Problem-A-Bit-Plus-Plus/BitPlusPlus.cpp
#include<iostream>
#include<string>

using namespace std;

int main () {
    int number;
    cin >> number;

    int result = 0;

    for (int i = 0; i < number; i++) {
        string str;
        cin >> str;

        if (str == "X++" || str == "++X") {
            result++;
        } else {
            result--;
        }
    }

    cout << result << endl;
}